i18n/lang.json

```diff
@@ -0,0 +1,543 @@
+{
+    "locale": "en-US",
+    "chartTitle": "Chart title",
+    "pieSliceName": "Slice",
+    "seriesName": "Series {add series.index 1}",
+    "yAxisTitle": "Values",
+    "rangeSelector": {
+        "allText": "All",
+        "allTitle": "View all",
+        "monthText": "{count}m",
+        "monthTitle": "View {count} {#eq count 1}month{else}months{/eq}",
+        "yearText": "{count}y",
+        "yearTitle": "View {count} {#eq count 1}year{else}years{/eq}",
+        "ytdText": "YTD",
+        "ytdTitle": "View year to date"
+    },
+    "weekFrom": "Week from",
+    "stockOpen": "Open",
+    "stockHigh": "High",
+    "stockLow": "Low",
+    "stockClose": "Close",
+    "viewFullscreen": "View in full screen",
+    "exitFullscreen": "Exit from full screen",
+    "printChart": "Print chart",
+    "downloadPNG": "Download PNG image",
+    "downloadJPEG": "Download JPEG image",
+    "downloadPDF": "Download PDF document",
+    "downloadSVG": "Download SVG vector image",
+    "contextButtonTitle": "Chart context menu",
+    "loading": "Loading...",
+    "months": [
+        "January",
+        "February",
+        "March",
+        "April",
+        "May",
+        "June",
+        "July",
+        "August",
+        "September",
+        "October",
+        "November",
+        "December"
+    ],
+    "shortMonths": [
+        "Jan",
+        "Feb",
+        "Mar",
+        "Apr",
+        "May",
+        "Jun",
+        "Jul",
+        "Aug",
+        "Sep",
+        "Oct",
+        "Nov",
+        "Dec"
+    ],
+    "weekdays": [
+        "Sunday",
+        "Monday",
+        "Tuesday",
+        "Wednesday",
+        "Thursday",
+        "Friday",
+        "Saturday"
+    ],
```

Comment:
Months, shortMonths and weekdays are now undefined by default, because those strings are picked from the `Intl` object, based on `lang.locale`. The same applies to `decimalPoint` and `thousandsSep`, which means those should be removed from the i18n JSON files.